specification/web/resource-manager/Microsoft.Web/stable/2016-06-01/logicAppsManagementClient.json

```diff
@@ -0,0 +1,2059 @@
+            "x-ms-azure-resource": true
+        },
+        "ConnectionGatewayInstallationDefinition": {
+            "description": "The gateway installation",
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "properties": {
+                    "$ref": "#/definitions/ConnectionGatewayInstallationPropertiesDefinition"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/ResourceDefinition"
+                }
+            ]
+        },
+        "ConnectionGatewayInstallationPropertiesDefinition": {
+            "description": "The gateway installation properties",
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "connectionGateway": {
+                    "$ref": "#/definitions/ConnectionGatewayReference"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/GatewayPropertiesDefinition"
+                }
+            ]
+        },
+        "ConnectionGatewayReference": {
+            "description": "The gateway reference",
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "name": {
+                    "description": "The resource reference name",
+                    "type": "string"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/ResourceReference"
+                }
+            ]
+        },
+        "CustomApiDefinition": {
+            "type": "object",
+            "description": "A custom API",
+            "additionalProperties": false,
+            "properties": {
+                "properties": {
+                    "$ref": "#/definitions/CustomApiPropertiesDefinition"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/ResourceDefinition"
+                }
+            ]
+        },
+        "CustomApiPropertiesDefinition": {
+            "type": "object",
+            "description": "Custom API properties",
+            "additionalProperties": false,
+            "properties": {
+                "connectionParameters": {
+                    "$ref": "#/definitions/InsensitiveDictionaryOfJToken"
+                },
+                "runtimeUrls": {
+                    "type": "array",
+                    "description": "Runtime URLs",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "capabilities": {
+                    "type": "array",
+                    "description": "The custom API capabilities",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "swagger": {},
```

Comment:
missing type (just object) and description

Reply:
Added